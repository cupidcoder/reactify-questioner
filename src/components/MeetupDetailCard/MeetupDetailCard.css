.MeetupDetailCard {
  display: flex;
  justify-content: space-between;
  width: 90%;
  margin: 0 auto;
  line-height: 1.5rem;
  border-bottom: 1px solid rgba(46,62,72,.12);
}

.EventDetail {
  padding: 1rem;
}

.EventDetail h2 {
  margin-bottom: 0.8rem;
}

.EventDetail p i {
  padding-right: 0.5rem;
  color: #305b8c;
}

.Description {
  padding: 1rem;
  border-left: 1px solid rgba(46,62,72,.12);
  width: 70%;
  text-align: justify;
}
