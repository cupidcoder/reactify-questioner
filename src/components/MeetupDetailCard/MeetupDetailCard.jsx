import React from 'react';
import style from './MeetupDetailCard.css';

const MeetupDetailCard = ({
  name, description, location, date, attending
}) => (
  <div className={style.MeetupDetailCard}>
    <div className={style.EventDetail}>
      <h2>{name}</h2>
      <p>
        <i className='fas fa-calendar-alt' />
        {date}
      </p>
      <p>
        <i className='fas fa-map-marker-alt' />
        {location}
      </p>
      <p>
        <i className='fas fa-users' />
        {attending}
      </p>
    </div>
    <div className={style.Description}>
      <h3>{description}</h3>
    </div>
  </div>
);

export default MeetupDetailCard;
