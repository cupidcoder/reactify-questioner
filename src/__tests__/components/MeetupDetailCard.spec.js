import React from 'react';
import { shallow } from 'enzyme';
import MeetupDetailCard from '../../components/MeetupDetailCard/MeetupDetailCard';

describe('MeetupDetailCard', () => {
  it('Should correctly save snapshot of the MeetupDetailCard', () => {
    const wrapper = shallow(<MeetupDetailCard />);
    expect(wrapper).toMatchSnapshot();
  });
});
