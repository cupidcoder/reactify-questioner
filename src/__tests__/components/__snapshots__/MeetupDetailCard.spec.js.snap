// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MeetupDetailCard Should correctly save snapshot of the MeetupDetailCard 1`] = `ShallowWrapper {}`;
